### ISSUE TITLE
Server isn't returning a JSONP response

### ISSUE THREAD

**Issue by Rodrigora** [blinktrade/frontend#236]
Hello Blinktrade,

I'm following your documentation and I've found this error when trying to access the public API:

```
import { BlinkTradeRest } from 'blinktrade';
const BlinkTrade = new BlinkTradeRest({
    currency: 'BRL',
    prod: false
  })
BlinkTrade.ticker().then(console.log)
```

![screen shot 2018-01-02 at 23 41 23](https://user-images.githubusercontent.com/371053/34506731-858361e6-f016-11e7-93d9-e1f8d1de99ff.png)

Look like the library is requiring a JSONP response, but the server returns a JSON response. 
Is it a bug in the library/server?

**Comment by pinhopro** [comment]
The server doesn't support jsonp

**Comment by Rodrigora** [comment]
Hi @pinhopro, thanks for your reply.

Yes, I see. 

So I don't understand why the Blinktrade library is requesting a JSONP format. I'll try again, thanks!

**Comment by cesardeazevedo** [comment]
Hi @Rodrigora 

We deprecated jsonp and we already ship a fix on the library (`0.0.17`), take a look at this issue for a better understanding (https://github.com/blinktrade/BlinkTradeJS/issues/32) and just update the library.

**Comment by Rodrigora** [comment]
Thanks guys! And sorry for creating this issue in the wrong repo
